src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/MauiCustomViewTarget.java

```diff
@@ -27,7 +27,7 @@ public MauiCustomViewTarget(@NonNull ImageView view, ImageLoaderCallback callbac
 
     @Override
     protected void onResourceCleared(@Nullable Drawable placeholder) {
-		this.view.setImageDrawable(placeholder);
+		post(() -> this.view.setImageDrawable(placeholder));
```

Comment:
Found the docs on this callback:

https://bumptech.github.io/glide/javadocs/490/com/bumptech/glide/request/target/CustomViewTarget.html

It doesn't say what thread this is on, but it must be the main thread because `view.setImageDrawable` would throw? 

Reviewing some examples on GitHub:

* https://github.com/CzBiX/v2ex-android/blob/f6a79a91b84ae8030930231d965845798c4ce0a8/app/src/main/java/com/czbix/v2ex/parser/AsyncImageGetter.java#L196-L198
* https://github.com/signalapp/Signal-Android/blob/2cef06cd6ef2ed70fdab3fcb525df9909f18aa5a/app/src/main/java/org/thoughtcrime/securesms/util/AvatarUtil.java#L71-L74

I don't think we need this `post()`, did it fix something when you added it?

Reply:
It did not fix anything specific, no. I think that, technically, only the ConfigureAwait(false) removal is needed. The post() calls were suggested changes that might help make MAUI more robust against this type of problem if customers did things on other threads.